StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,109 @@
+using Mars_Luiz.Pages;
+using OpenQA.Selenium.Chrome;
+using System;
+using TechTalk.SpecFlow;
+using Mars_Luiz.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using NUnit.Framework.Legacy;
+
+namespace Mars_Luiz.StepDefinitions
+{
+    [Binding]
+    public class LanguageStepDefinitions :CommonDriver
+    {
+        [Given(@"I logged into the portal successfull")]
+        public void GivenILoggedIntoThePortalSuccessfull()
+        {
+            //Open chrome browser
+            driver = new ChromeDriver();
+            driver.Manage().Window.Maximize();
+            //Sign IN page object initialization and definition
+            SignInPage signInPageObj = new SignInPage();
```

Comment:
you should move your objects out of the steps and initialise them in the constructor, please see
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors